pkg/monitoring/vms/prometheus/prometheus.go

```diff
@@ -47,6 +48,17 @@ var (
 		nil,
 	)
 
+	// higher-level, telemetry-friendly metrics
+	vmiCountDesc = prometheus.NewDesc(
+		"kubevirt_vmi_status_count",
+		"VMI status.",
+		[]string{
+			"node", "status",
```

Comment:
I think you meant `phase` and not `status`.

Reply:
yep, good catch